src/main/java/com/mraof/minestuck/item/crafting/alchemy/GristSet.java

```diff
@@ -27,8 +27,15 @@
 	
 	public static final GristSet EMPTY = new GristSet(Collections.emptyMap());
 
+	/**
+	this has been replaced to allow the GristGutter to have full access to this dictionary
+	 */
+	/*
 	private final Map<GristType, Long> gristTypes;
-
+	 */
```

Comment:
I think these comments can be safely removed

Reply:
done, how do i summon a pul request